cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -11,18 +13,23 @@ var EventObserver = require( '../modules/util/EventObserver' );
  *
  * @param {HTMLNode} element
```

Comment:
hm, good point. Toggle was added in prep for updating the state in the mega menu across the three flyout menus it contains. When going from the first to second or second to third menu, the parent menu is still expanded, because it contains the child. So I added toggle as an event so that the mega menu uses it to update the transitions based on a change in the menu, whether that's an expand or collapse. But you're right, that complexity doesn't need to sit in the FlyoutMenu and could go into the MegaMenu itself, which has an internal _toggle method it calls on either an expand or collapse of its menus.